core/domain/base_model_validators.py

```diff
@@ -31,7 +31,7 @@
 (base_models,) = models.Registry.import_models([models.NAMES.base_model])
 datastore_services = models.Registry.import_datastore_services()
 
-USER_ID_REGEX = 'uid_[a-z]{32}|OppiaSuggestionBot|OppiaMigrationBot'
+USER_ID_REGEX = 'uid_[a-z]{32}'
```

Comment:
Removed these since no such models should be created now (we can remove the existing ones):

- OppiaMigrationBot is used only for editing explorations and edit is not recorded if made by a bot: https://github.com/oppia/oppia/blob/develop/core/domain/exp_services.py#L945
- OppiaSuggestionBot is only used for [rejecting suggestions](https://github.com/oppia/oppia/blob/develop/core/domain/suggestion_services.py#L446) and there is no update to any user model using the bot id while rejecting suggestion.